platform/broadcom/rules.mk

```diff
@@ -33,7 +35,7 @@ SONIC_ALL += $(SONIC_ONE_IMAGE) $(SONIC_ONE_ABOOT_IMAGE) \
              $(DOCKER_FPM)
 
 # Inject brcm sai into syncd
-$(SYNCD)_DEPENDS += $(BRCM_SAI) $(BRCM_SAI_DEV)
+$(SYNCD)_DEPENDS += $(BRCM_SAI) $(BRCM_SAI_DEV) $(BRCM-DNX_SAI)
```

Comment:
move it between $(BRCM_SAI) and $(BRCM_SAI_DEV) , rename to BRCM_DNX_SAI

Reply:
Sure will update.